tests/mentorship_relation/test_api_send_request.py

```diff
@@ -62,9 +72,45 @@ def test_fail_send_request_bad_user_id(self):
                                            headers=auth_header, content_type='application/json',
                                            data=json.dumps(test_payload))
         self.assertEqual(400, actual_response.status_code)
+	self.assertEqual(1, len(outbox))
```

Comment:
is this well indented?

Reply:
@isabelcosta , I have made the required indentation changes after removing unnecessary commits, please review and suggest changes if any.